pkg/sql/parser/builtins.go

```diff
@@ -246,6 +247,33 @@ var Builtins = map[string][]Builtin{
 		},
 	},
 
+	"from_ip": {
+		Builtin{
+			Types:      ArgTypes{TypeBytes},
+			ReturnType: TypeString,
+			fn: func(_ *EvalContext, args DTuple) (Datum, error) {
+				ipstr := args[0].(*DBytes)
+				nboip := net.IP(*ipstr)
+				sv := nboip.String()
+				if sv == "<nil>" {
```

Comment:
Can you add a comment here that explains in which circumstances "<nil>" is returned.